usaspending_api/idvs/v2/views/amounts.py

```diff
@@ -9,12 +10,167 @@
 
 from usaspending_api.awards.models import ParentAward
 from usaspending_api.common.cache_decorator import cache_response
+from usaspending_api.common.helpers.sql_helpers import execute_sql_to_ordered_dictionary
 from usaspending_api.common.validator.award import get_internal_or_generated_award_id_model
 from usaspending_api.common.validator.tinyshield import TinyShield
 
 
 logger = logging.getLogger("console")
 
+defc_sql = """
```

Comment:
How similar is this SQL to the new SQL in usaspending_api/awards/v2/data_layer/orm.py? Could they be merged into a helper function?

Reply:
they're essentially the same, the only real difference is that the idv sql queries against a list of award_ids, which is easy enough to change